DataFormats/Headers/src/DataHeader.cxx

```diff
@@ -27,14 +27,15 @@ const o2::Header::SerializationMethod o2::Header::gSerializationMethodFlatBuf("F
 //__________________________________________________________________________________________________
 //possible data origins
 const o2::Header::DataOrigin o2::Header::gDataOriginAny    ("***");
-const o2::Header::DataOrigin o2::Header::gDataOriginInvalid("   ");
+const o2::Header::DataOrigin o2::Header::gDataOriginInvalid("INV");
```

Comment:
I disagree. If I then printfs something which includes gDataOriginInvalid, the string will be truncated or empty. That's actually why I changed it (I was debugging things and the message looked ok when I realised something was not getting printed).